### PR TITLE
Set default module root to cwd

### PR DESCRIPTION
**Pull request by pluma**
### Scope & Purpose

This prevents accidental traversal to the file system root when attempting to resolve module paths while using arangosh. We already set the module root for Foxx services but left it unbounded by default so arangosh wouldn't know when to stop searching. This can have unexpected results when a node_modules folder is present at a higher level of hierarchy.

- [x] :hankey: Bugfix (requires CHANGELOG entry)

#### Backports:

- [x] No backports required

### CODE REVIEW

**Comment by pluma** [comment]
@jsteemann does this fix the problem you encountered?

**Comment by pluma** [comment]
Test failure seems to be unrelated:

```
* Test "server_permissions"
    [FAILED]  tests/js/client/server_permissions/test-oasis.js

      "testStatisics" failed: Error: at assertion #1: assertTrue: (false) does not evaluate to true
```

File: CHANGELOG [hunk 1]
```diff
@@ -6036,6 +6036,7 @@ devel
 
 * Fixed an agency bug found in Windows tests.
 
+* Limited module resolution in arangosh to the path from which arangosh is invoked.
```
Comment (jsteemann) [inline CHANGELOG:6039]: Please move this entry to the top of the CHANGELOG, so it becomes the "newest" entry in the "devel" section.

**Comment by pluma** [comment]
https://jenkins.arangodb.biz/job/arangodb-matrix-pr/19801/

**Review by jsteemann — APPROVED** [review]
LGTM!

**Pull request merged**